hs_core/hydroshare/hs_bagit.py

```diff
@@ -90,24 +80,23 @@ def create_bag_files(resource, fed_zone_home_path=''):
     # istorage.saveFile('', to_file_name, create_directory=True)
 
     # create resourcemetadata.xml and upload it to iRODS
-    from_file_name = '{path}/resourcemetadata.xml'.format(path=bagit_path)
+    from_file_name = os.path.join(temp_bagit_path, 'resourcemetadata.xml')
     with open(from_file_name, 'w') as out:
         # resources that don't support file types this would write only resource level metadata
         # resource types that support file types this would write resource level metadata
         # as well as file type metadata
-        out.write(resource.get_metadata_xml())
-    to_file_name = '{res_id}/data/resourcemetadata.xml'.format(res_id=resource.short_id)
-    if fed_zone_home_path:
-        to_file_name = '{fed_zone_home_path}/{rel_path}'.format(
-                            fed_zone_home_path=fed_zone_home_path,
-                            rel_path=to_file_name)
+        out.write(resource.metadata.get_xml())
```

Comment:
In that case keep the original statement:
``` out.write(resource.get_metadata_xml()) ```

Resource and file level metadata is written to the same xml file.